### ISSUE TITLE
overload list

### ISSUE THREAD

**Issue by beckmi** [cubei/flashcards#16]
If you add a list, which was imported before, it is listed twice. Only closing and reopening flashcards removes the first instance from the list.

**Comment by cubei** [comment]
Yes, but it's not necessary to completely close the app, going back one activity and the reopen the list again should also fix it.
